fix: pruning-at-boot default (#200)
Consensys/doc.web3signer: docs/reference/cli/subcommands.md

# Example

```bash
--slashing-protection-pruning-at-boot-enabled=true
```

# Environment variable

```bash
WEB3SIGNER_ETH2_SLASHING_PROTECTION_PRUNING_AT_BOOT_ENABLED=true
```

# Configuration file

```yaml
eth2.slashing-protection-pruning-at-boot-enabled: true
```

<!--/tabs-->

When set to `true`, [slashing protection database
pruning](../../how-to/configure-slashing-protection.md#prune-the-slashing-protection-database) is

> enabled at boot as well as in the defined interval [pruning intervals](#slashing-protection-pruning-interval).

Reviewer: ```suggestion
enabled at startup and at the defined [pruning intervals](#slashing-protection-pruning-interval).
```
